components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/InMemoryAPIDeployer.java

```diff
@@ -120,19 +121,20 @@ public boolean deployAllAPIsAtGatewayStartup (Set<String> assignedGatewayLabels)
                         for (String APIruntimeArtifact :gatewayRuntimeArtifacts){
                             GatewayAPIDTO gatewayAPIDTO = null;
                             try {
-                                if (APIruntimeArtifact != null) {
+                                if (!StringUtils.isEmpty(APIruntimeArtifact)) {
```

Comment:
fix variable naming and same goes here

Reply:
Fixed f1fefa5ebc85177015c7bdac17899ebac013acdd